docs/guide/timeseries_metadata.rst

```diff
@@ -0,0 +1,87 @@
+.. _timeseries-metadata-explanation:
+
+A Detailed Look at the Metadata
+===============================
+
+TimeSeries store metadata in a `~sunpy.timeseries.TimeSeriesMetaData` object, this object is designed to be able to store multiple basic `~sunpy.util.metadata.MetaDict` (case-insensitive ordered dictionary) objects and able to identify the relevant metadata for a given cell in the data.
+This enables a single TimeSeries to be created by combining/concatenating multiple TimeSeries source files together into one and to keep a reliable track of all the metadata relevant to each cell, column or row.
+The metadata can be accessed by:
+
+.. code-block:: python
+
+    >>> meta = my_timeseries.meta # doctest: +REMOTE_DATA
```

Comment:
How does this work? I thought it had to be defined in the same rst page.

Reply:
Oops it doesn't work. The test fails.